Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/analysis/ScalarOperandAnalyzer.java

```diff
@@ -154,9 +154,8 @@ void checkOperands(Program program, Instruction instr) {
 					// don't do any addresses that could be numbers, even if they are in the
 					//   address space.
 					long value = scalar.getUnsignedValue();
-					if (value < 4096 || value == 0xffff || value == 0xff00 || value == 0xffffff ||
-						value == 0xff0000 || value == 0xff00ff || value == 0xffffffff ||
-						value == 0xffffff00 || value == 0xffff0000 || value == 0xff000000) {
+					if (value < 4096 || value == 0xffff || value == 0xff00 || value == 0xffffff
+							|| value == 0xff0000 || value == 0xff00ff) {
```

Comment:
Actually not the middle row but the last 4 "clauses" have been removed because they are comparing "value" to another number, which is negative. If value was negative, the whole condition is true anyway because of the first statement: `value < 4096`

![image](https://user-images.githubusercontent.com/29959150/79092742-1fa63d00-7d52-11ea-9f2c-eeea8043ecd9.png)

Feel free to correct me if I'm wrong, but that's what I was thinking...